src/ClientBuilder.php

```diff
@@ -141,15 +107,23 @@ public function __construct(array $options = [])
     /**
      * {@inheritdoc}
      */
-    public static function create(array $options = []): self
+    public static function create(Options $options = null): ClientBuilderInterface
```

Comment:
My opinion on this is twofold.
I would leave `create(array $options = [])` but in the ctor we should use `__construct(Options $options = null)` and create the options object in `create.

Reply:
We can also do that, I'm not against this as long as there is a way to create the builder from a simple array of options so that the user won't need to instantiate himself another object